examples/trusted_database/oak_app_manifest.toml

```diff
@@ -0,0 +1,5 @@
+name = "trusted_database"
+
+[modules]
+app = { path = "examples/trusted_database/bin/trusted_database.wasm" }
+trusted_database_handler = { path = "examples/trusted_database/bin/trusted_database_handler.wasm" }
```

Comment:
I guess I don't quite understand the plumbing here. Do you still need this here? Is not it enough that `TrustedDatabaseNode` is creating `trusted_database_handler` nodes per request? 
The same applies to the integration_test for `module_0`. Do you need to build both wasm modules for the test?

Reply:
`oak_app_manifest.toml` is used by `oak_loader` to load corresponding Wasm modules. So we still need to specify all possible modules here.